src/pages/resources/acp-end-of-support.md

```diff
@@ -1,5 +1,21 @@
 # ACP SDKs end-of-support
```

Comment:
Since we expanded the scope of this file, should it be `End-of-support FAQ`  or similar? (same applies for the gatsby config file)

Reply:
We can do it, but that would break this link.

Reply:
couple options:
1. set up a redirect
2. don't rename the file, just the titles

Reply:
cc @calebk1 this page got renamed to sdks-end-of-support, can you help with a redirect?